spec.html

```diff
@@ -13624,9 +13642,21 @@ <h2>Syntax</h2>
       <emu-note>
         <p>For example: `3in` is an error and not the two input elements `3` and `in`.</p>
       </emu-note>
-      <p>A conforming implementation, when processing strict mode code, must not extend, as described in <emu-xref href="#sec-additional-syntax-numeric-literals"></emu-xref>, the syntax of |NumericLiteral| to include <emu-xref href="#prod-annexB-LegacyOctalIntegerLiteral"></emu-xref>, nor extend the syntax of |DecimalIntegerLiteral| to include <emu-xref href="#prod-annexB-NonOctalDecimalIntegerLiteral"></emu-xref>.</p>
 
-      <emu-clause id="sec-static-semantics-mv">
+      <emu-clause id="sec-numeric-literals-early-errors">
+        <h1>Static Semantics: Early Errors</h1>
+        <emu-grammar>
+          NumericLiteral :: LegacyOctalIntegerLiteral
+
+          DecimalIntegerLiteral :: NonOctalDecimalIntegerLiteral
+        </emu-grammar>
+        <ul>
+          <li>It is a Syntax Error if the source code matching this production is strict mode code.</li>
+        </ul>
+        <emu-note>In non-strict code, this syntax is allowed, but deprecated.</emu-note>
```

Comment:
I don't think we use the term "deprecated". Should this be marked "legacy" as we've defined it in #2125?

Reply:
From editor call today, we will request committee consensus to apply the term "legacy" here.

Reply:
Given that these already had "legacy" in their names, do we actually need consensus for this, do you think?

Reply:
We have consensus to mark these as legacy.

Reply:
So I should just change "deprecated" to "legacy", or what wording would you like?